R-package/man/xgb.train.Rd

```diff
@@ -210,7 +211,7 @@ dtest <- xgb.DMatrix(agaricus.test$data, label = agaricus.test$label)
 watchlist <- list(train = dtrain, eval = dtest)
 
 ## A simple xgb.train example:
-param <- list(max_depth = 2, eta = 1, verbosity = 0, nthread = 2, 
+param <- list(max_depth = 2, eta = 1, silent = 1, nthread = 2,
```

Comment:
The silent parameter is deprecated, see https://xgboost.readthedocs.io/en/latest/parameter.html .

Reply:
@trivialfis oh this is interesting. I didn't actually touch this line directly. File changes in `.Rd` files were auto-generated when I ran `devtools::document()`.

Maybe I should rebase and try again?

Reply:
ohhhhh I wonder if someone in a previous PR manually edited the `.Rd` but not the corresponding roxygen block in the R code? I'll fix that on this PR

Reply:
Ok yes, this is what happened! I walked through the blame and found that this parameter was changed in `xgb.train.Rd` in #4267 but not the corresponding roxygen content in the R source code.

Reply:
fixed in https://github.com/dmlc/xgboost/pull/4345/commits/25dbe3a9ccd6ed0d5d08c5a9c459a46d33b2ec1b

Reply:
@jameslamb Thanks for fixing my mistake!  I restarted the test and will merge once they are done.

Reply:
No problem, happy to help!